frontend/src/diagram/sprotty/views/LabelView.tsx

```diff
@@ -48,12 +49,15 @@ export class LabelView extends SLabelView {
         styleObject['text-decoration'] += ' line-through';
       }
     }
+    if (label.type?.includes('center')) {
```

Comment:
Can the type really be null? Does that mean that the backend does not have the precise position of the label anymore? For example to compute overlap and things like that.